dotnet/src/Connectors/Connectors.HuggingFace/HuggingFaceKernelBuilderExtensions.cs

```diff
@@ -4,8 +4,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.SemanticKernel.AI.Embeddings;
 using Microsoft.SemanticKernel.AI.TextGeneration;
```

Comment:
Were we going to get rid of these ".AI."s as well?

Reply:
IN the other PR 😊